content/news/buzz/march-2025-status-update/contents.lr

```diff
@@ -0,0 +1,64 @@
+
+* We `modified the way that style properties are applied to Toga widgets <https://github.com/beeware/toga/pull/3273>`__. Style updates are now applied in batch updates, dramatically reducing the amount of repeated work required when changing the appearance of a widget. 
+
+* We `improved the UX of the Winforms Table widget <https://github.com/beeware/toga/pull/3293>`__, ensuring that the selected row is still visible when a table doesn't have focus.
+
+* We improved Toga's `handling of the asyncio event loop on Windows <https://github.com/beeware/toga/pull/3270>`__. This change ensures that Winforms apps always clean up the event loop on exit; previously, it was possible for warnings to be raised on application exit, caused by open sockets and unawaited co-routines.
+
+* We `modified the handling of alignment style properties <https://github.com/beeware/toga/pull/3268>`__ to ensure that text *direction* is changed, the layout of the app is re-evaluated.
+
+* We `added support for controlling inter-line spacing in multiline text on a canvas <https://github.com/beeware/toga/pull/3217>`__.
+
+* We `corrected some issues with background color handling on Android <https://github.com/beeware/toga/pull/3118>`__.
+
+* We `corrected some issues with supporting PyGObject on Linux distributions that are based on Debian 12 (including Ubuntu 22.04) <https://github.com/beeware/toga/pull/3256>`__. This was necessary because of a backwards incompatible change introduced into the PyGObject 3.52.1 release; unfortunately, users of older Linux distributions may need to make some modifications to their projects.
+
+* The pull request `adding iOS support to cibuildwheel <https://github.com/pypa/cibuildwheel/pull/2286>`__ has been merged. With this change, a simple package can build binary wheels for iOS with a single command: ``CIBW_PLATFORM=ios cibuildwheel``. This PR then `required a follow-up investigation and fix <https://github.com/pypa/cibuildwheel/pull/2337>`__ to correct an intermittent issue that was seen when running the cibuildwheel tests in CI.
+
+* We've started work on a follow-up PR to cibuildwheel, `adding support for explicitly declaring safe cross-build tools <https://github.com/pypa/cibuildwheel/pull/2317>`__. This is a use case that was identified during the development of PR for iOS support.
+
+* We corrected a `long standing issue with PTH file handling in Briefcase <https://github.com/beeware/briefcase/pull/2212>`__. 
+
+* We added the ability to invoke ``briefcase build`` and ``briefcase package`` on `a single application in a Briefcase project <https://github.com/beeware/briefcase/pull/2201>`__.
+
+* We began work on a patch to support generating iOS binary wheels for NumPy. This is a work in progress; we've got a set of patches that allows us to *compile* NumPy, but the test suite won't run yet.
```

Comment:
```suggestion
* We began work on a patch to support generating iOS binary wheels for NumPy. This is a work in progress; we've got a set of patches that allows us to *compile* NumPy, but the test suite won't run yet.

* We created a process to make an official `binary release of Python for Android <https://github.com/python/cpython/pull/131532>`__, which should be available on python.org starting with Python 3.14. This will reduce duplicated effort across Python on Android tools, and ensure binary compatibility in the packaging ecosystem.

* We submitted pull requests adding Android support to `packaging <https://github.com/pypa/packaging/pull/880>`__ and `pip <https://github.com/pypa/pip/pull/13303>`__.
```